### PR TITLE
Blog-App Add forms

### PR DESCRIPTION
**Pull request by Kossi-stack**
**In this pull request these are the following requirement**

- Created form to create a post on behalf of the current_user 
- Created form to create a comment on behalf of the current_user

### CODE REVIEW

File: app/controllers/posts_controller.rb [hunk 1]
```diff
@@ -7,4 +8,21 @@ def index
...
+
+  def new
+    @post = Post.new
+  end
+
+  def create
+    @post = Post.new(params[:id])
+    @post.title = params[:title]
+    @post.text = params[:text]
+    @post.user_id = current_user.id
```
Comment (mke2111) [inline app/controllers/posts_controller.rb:20]: - [x] Just like in the comments controller, Add post_params as a private method which you will then pass to the create action. [Use this lesson as a recap](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/lessons/Controllers_lesson.md#:~:text=Pay%20special%20attention%20to%20the%20strong%20parameters%20description%20in%20Rails%20Guides).

**Review by mke2111 — CHANGES_REQUESTED** [review]
### Hey @Kossi-stack, 

**This is a great project you have going on, well-done lad**. 

   
**However, there are a few sections to improve before your project is approved.** 🔴 
<hr>

## Required Changes ♻️

- [ ] A user should be able to add a like to posts. Add the likes controller which should have a create action. Also, remember to add the necessary routes for the likes action.

_Check the inline comments under the review for sections to improve._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/controllers/posts_controller.rb (hunk 1 above)
Comment (Kossi-stack) [reply app/controllers/posts_controller.rb:20]: Thanks, I just fix it

**Review by rukundoeric — APPROVED** [review]
Hi @Kossi-stack,

_Excellent job on this project. Keep it up 🎆_

_Congratulations! 🎉  on the splendid job 👍
You can now merge this PR and submit the project completion form 🚀 . I wish you all the very best in your next tasks 🥇_


Cheers and Happy coding!👏👏👏

__Feel free to leave any questions or comments in the PR thread if something is not 100% clear.__

------
>As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**